### PR TITLE
Implement Dataloader for PhoATIS Intent Classification

### PR DESCRIPTION
**Pull request by dehanalkautsar**
#139 . (Not closed as this is only one of two tasks in the dataset).

### Note
- Add some intent labels (i.e. `day_name`, `aicraft#flight`, `airline#flight`, `airfare#flight_time`) that haven't appeared in the datasets' constant label, but occur in some examples.
- Add `Slot Filling` task in the SEACrowd schema

### Checkbox
- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `seacrowd/sea_datasets/my_dataset/my_dataset.py` (please use only lowercase and underscore for dataset naming).
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_SEACROWD_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_seacrowd seacrowd/sea_datasets/<my_dataset>/<my_dataset>.py`.
- [ ] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

File: seacrowd/sea_datasets/phoatis_intent_cls/phoatis_intent_cls.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+            data_intent = fw.read()
+            data_intent = data_intent.split("\n")
+
+        if self.config.schema == "source":
+            for idx, text in enumerate(data_input):
+                example = {}
+                example["id"] = str(idx)
+                example["text"] = text
+                example["intent_label"] = data_intent[idx]
+                data_slot[idx] = data_slot[idx].split()
```
Comment (gentaiscool) [inline seacrowd/sea_datasets/phoatis_intent_cls/phoatis_intent_cls.py:192]: I think you don't need this line. You can split the data_slot on Line 193

**Review by gentaiscool — CHANGES_REQUESTED** [review]
minor changes required

**Comment by dehanalkautsar** [comment]
Hi @gentaiscool, I have fixed the minor changes that are required.

**Comment by sabilmakbar** [comment]
Hi @holylovenia @gentaiscool, since this is a multitask dataset and there's another PR of #235 for the same dataset, I'm thinking of unifying it rather than having 2 different dataloader scripts. 

Fortunately, both the assignee and the reviewers are the same. wdyt?

**Comment by dehanalkautsar** [comment]
> Hi @holylovenia @gentaiscool, since this is a multitask dataset and there's another PR of #235 for the same dataset, I'm thinking of unifying it rather than having 2 different dataloader scripts.
> 
> Fortunately, both the assignee and the reviewers are the same. Should we close one of the PRs and discuss on the other one? wdyt?

I agree with that. Should we close this PR and discuss it on the other one?

**Comment by holylovenia** [comment]
@sabilmakbar @dehanalkautsar I agree since PhoATIS Intent Classification and PhoATIS Slot Filling should be unified as a PhoATIS dataloader according to #139. I'll close this PR. Let's move to #235.

cc: @gentaiscool

**Pull request closed without merging**